templates/.vscode/tasks.json

```diff
@@ -142,9 +142,9 @@
         {
             "label": "Monitor: Start the monitor",
             "type": "shell",
-            "command": "${config:idf.pythonBinPath} ${config:idf.espIdfPath}/tools/idf.py -p ${config:idf.port} -b ${config:idf.baudRate} monitor",
+            "command": "${config:idf.pythonBinPath} ${config:idf.espIdfPath}/tools/idf.py -p ${config:idf.port} monitor",
             "windows": {
-                "command": "${config:idf.pythonBinPathWin} ${config:idf.espIdfPathWin}\\tools\\idf.py monitor -p ${config:idf.portWin} -b ${config:idf.baudRate}"
+                "command": "${config:idf.pythonBinPathWin} ${config:idf.espIdfPathWin}\\tools\\idf.py monitor -p ${config:idf.portWin}"
```

Comment:
maybe we need to pass the monitor baud rate

Reply:
Based on @igrr comment, `idf.py monitor` already picks the value from CONFIG_ESPTOOLPY_MONITOR_BAUD in https://github.com/espressif/esp-idf/blob/8bc19ba893e5544d571a753d82b44a84799b94b1/tools/idf_py_actions/serial_ext.py#L87-L88

Reply:
I see maybe add a doc explaining this also add the priority/preference order of the baud rates for the monitor.